encryption/src/main/java/com/novoda/demo/encryption/EncryptionActivity.java

```diff
@@ -14,16 +22,12 @@
 import javax.crypto.SecretKeyFactory;
 import javax.crypto.spec.PBEParameterSpec;
 
-import android.app.Activity;
-import android.os.Bundle;
-import android.util.Log;
-import android.widget.TextView;
-
-public class Encrypt extends Activity {
+public class EncryptionActivity extends Activity {
 
-	private static final String SECRET_PASSWORD_TO_ENCRYPT = "secretPassword";
+	private static final String SECRET_DATA_TO_ENCRYPT = "secretPassword";
 	private static final String TAG = "Encrypt";
-	
+    private static final String PREFERENCE_KEY_SECURED_DATA = "KEY_SECURED_DATA";
+
     final String CIPHER_TYPE = "PBEWithMD5AndDES/CBC/PKCS5Padding";
     final String ALGORITHM = "PBEWithMD5AndDES";
```

Comment:
Aren't better algorithms available on Android? SunJCE doesn't have them AFAIR. But BouncyCastle should have for example "PBEWITHSHA256AND128BITAES-CBC-BC" MD5 and DES are both outdated and considered insecure.